### PR TITLE
Manager portal localization

### PR DESCRIPTION
**Pull request by xander-miller**

### CODE REVIEW

File: resources/lang/en/manager/application_post.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    'header' => [
+        'twitter_title' => 'Visit :name\'s Twitter profile.',
+        'linkedin_title' => 'Visit :name\'s LinkedIn profile.'
+    ],
+    'content' => [
+        'job_poster' => 'Job Poster',
+        'review' => [
+            'name' => 'Name',
+            'info' => 'Info',
+            'reviewed' => 'Reviewed?tasitc',
```
Comment (Khristophor) [inline resources/lang/en/manager/application_post.php]: Looks like a typo here and on line 28.

Comment (xander-miller) [reply resources/lang/en/manager/application_post.php]: Yeah, that wasn't suppose to make it into the commit.

File: app/Http/Controllers/JobController.php [hunk 2]
```diff
@@ -248,7 +248,11 @@ function ($skillLevel) use ($skillLangs) {
         return view(
             'manager/job_create',
             [
-                'job_heading' => Lang::get($jobHeading),
+                /*Localization Strings*/
+                'job_l18n' => Lang::get('manager/job_create'),
```
Comment (Khristophor) [inline app/Http/Controllers/JobController.php]: This is incredibly nitpicky, but `i18n` is for internationalization, and `l10n` is for localization.

Comment (xander-miller) [reply app/Http/Controllers/JobController.php]: I :heart: :nerd: nitpicky! I will fix it in a commit tomorrow 😄 .

**Comment by xander-miller** [comment]
@gobyrne  Addition french localization file:

- [ ] resources/lang/fr/manager/job_create.php

**Comment by xander-miller** [comment]
@gobyrne Additional french localization file:

- [ ] resources/lang/fr/manager/job_index.php

File: routes/web.php [hunk 3]
```diff
@@ -217,7 +217,7 @@ function () : void {
                     "hero" => [
                         "hero_logo" => "/images/logo_tc_colour.png",
                         "hero_logo_alt" => "The GC Talent Cloud Graphic Identifier.",
```
Comment (Khristophor) [inline routes/web.php]: This should probably be localized as well.

File: resources/lang/fr/manager/job_create.php [hunk 4]
```diff
@@ -3,29 +3,93 @@
...
+
+
+
+
+
+
+
         ]
-    ],
-    'questions' => [
```
[the remark this answers is not in the document]
Comment (xander-miller) [reply resources/lang/fr/manager/job_create.php]: Here are the translations for the controller function that adds default questions. 205601f

File: routes/web.php (hunk 3 above)
Comment (xander-miller) [reply routes/web.php]: hero alt text localization a1d30b5

**Comment by xander-miller** [comment]
@gobyrne There is a new line in the home hero localization that needs translation:
- [ ]  /resources/lang/fr/manager/home_hero.php

**Pull request merged**